Reflections/Week_5_Reflection.md

```diff
@@ -0,0 +1,22 @@
+# Weekly Reflection- Week Five
+
+1. What have you accomplished this week? (Please list each accomplishment, and explain)
+  * I have finished the interiors for the mod. I have made the interiors accessible in game. I have finished navmeshing (movement pathing for AI) for the interiors. I have finished the companion perks that are rewarded on completion of the quest. I have made the quest finishable. It is important to note that certain things are going to have to get cut for the sake of time. I have decided that voiced NPCs are too time intensive to be done. Also, I have scrapped the dialogue between Mario and his boss for the same reason.
+
+![Main Office](WeeklyScreenshots/WeekFiveMainOffice.png)
+![Mario's Office](WeeklyScreenshots/WeekFiveMariosOffice.png)
+![Mario Fired Perk](WeeklyScreenshots/WeekFiveMarioFiredPerk.png)
+![Mario Promotion Perk](WeeklyScreenshots/WeekFiveMarioPromotionPerk.png)
+![Return Notes One](WeeklyScreenshots/WeekFiveReturnNotesOne.png)
+![Return Notes Two](WeeklyScreenshots/WeekFiveReturnNotesTwo.png)
+![Return Notes Three](WeeklyScreenshots/WeekFiveReturnNotesThree.png)
+
+2. What challenges or difficulties did you face? If you solved them, how? If not, what have you learned so far? Have you sought help or other resources?
+  * I have had to begin preparing to move out at the end of the semester. I have been working on a paper due this Friday.
```

Comment:
Very fair and understandable blockers here. Post-semester/college preparation is definitely rough.